### ISSUE TITLE
Add ability to log request content to AI

### ISSUE THREAD

**Issue by alexkarcher-msft** [Azure/azure-functions-host#1935]
Currently there is no setting in Functions or Proxies to log the request content in Application Insights. This content is currently logged in the standard Functions request logs, as well as across other services.

It is a security concern to have them logged by default, so a user-configurable setting should be added to toggle AI request content logging.

**Comment by mathewc** [comment]
I don't believe we're logging request details by default as you say - where do you see this? Also not HTTP specific - you'd want for other triggers as well.

**Comment by mattchenderson** [comment]
There are no plans to further enhance Proxies. Please see https://azure.microsoft.com/updates/?id=community-support-for-azure-functions-proxies-will-end-on-30-september-2025.
